ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-periodics.yaml

```diff
@@ -0,0 +1,289 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 0 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: redhat-developer
+    repo: gitops-operator
+  labels:
+    ci-operator.openshift.io/variant: v4.12
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-developer-gitops-operator-master-v4.12-periodic-kuttl-parallel
+  reporter_config:
+    slack:
+      channel: '#gitops-dev-qe'
```

Comment:
Are we okay using gitops-dev-qe?

Reply:
Sorry! I meant to add **#gitops-test-notification**